.gitlab-ci.yml

```diff
@@ -147,7 +147,7 @@ test-linux-nightly:
 
 build-android:
   <<:                              *build-on-linux
-  image:                           parity/rust-parity-ethereum-android-build:stretch
+  image:                           parity/parity-ci-android:stretch
```

Comment:
this one doesn't seem to exist, yet.
https://hub.docker.com/r/parity/parity-ci-android

Reply:
(haven't checked the others)

Reply:
Yup, thanks for vigilance, forgot to push it.
Just for sanity, listing all our images used there (in the order of appearance):
https://hub.docker.com/r/parity/parity-ci-linux/tags
https://hub.docker.com/r/parity/parity-ci-i386/tags
https://hub.docker.com/r/parity/parity-ci-arm64/tags
https://hub.docker.com/r/parity/parity-ci-armhf/tags
https://hub.docker.com/r/parity/awscli/tags
https://hub.docker.com/r/parity/parity-ci-docs/tags

Reply:
and pushed: https://hub.docker.com/r/parity/parity-ci-android/tags